static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -10,7 +10,7 @@ import {Organization} from 'sentry/types';
 import {DragManagerChildrenProps} from './dragManager';
 import {ScrollbarManagerChildrenProps, withScrollbarManager} from './scrollbarManager';
 import SpanBar from './spanBar';
-import SpanGroupBar from './spanGroupBar';
+import SpanGroupBar from './spanDescendantGroupBar';
```

Comment:
The import should be renamed so it's clearer.

Reply:
Ah, I see now why default exports are a problem. I was using automatic imports so I didn't notice this